android/src/main/java/io/branch/rnbranch/RNBranchModule.java

```diff
@@ -106,8 +106,9 @@ public String getName() {
   }
 
   @ReactMethod
-  public void getInitSessionResult(Promise promise) {
+  public void redeemInitSessionResult(Promise promise) {
```

Comment:
`getInitSessionResult` was an "internal" api, i.e. not exposed via the javascript module. 

While it is technically possible to access via `NativeModules.RNBranch.getInitSessionResult` I would say it is unlikely anyone has done this, and additionally it would be considered unsupported module usage: subject to change.